deployments/fuji/usdc/configuration.json

```diff
@@ -21,7 +21,6 @@
     "baseBorrowSpeed": "0.0011458333333333333e15",
     "baseMinForRewards": "1000000e6"
   },
-  "rewardToken": "WAVAX",
```

Comment:
Note: this is because scens are correctly failing to 'complete' the deploy, as this is not the currently configured rewards token in the roots contracts, and the normal signer cannot make the change (for an upgrade we would impersonate, but not for the 'initial' deploy). I believe the correct thing to do would be either (a) create a migration with the change, then possibly updating `configuration.json` or (b) wipe the `roots.json` and do a redeploy (which we should probably do now).

Reply:
Probably redeploy right? I think you're doing that with the new gov anyways?

Reply:
Yeah I did redeploy I guess, let's worry about realistic `fuji` settings later though, we can easily redeploy again (or migrate)